recipe/meta.yaml

```diff
@@ -68,7 +68,7 @@ requirements:
     - hdf5  # exact pin handled through hdf5 run_exports
     - libgdal >=3.6.2,<3.7.0a0
     - libkml >=1.3.0,<1.4.0a0
-    - libpq >=12.15,<13.0a0
+    - libpq >=17.0,<18.0a0
```

Comment:
pin_compatible?

Reply:
Seems the pattern in this recipe is to pin between a minimum acceptable and the next minor version. Does it make more sense here to convert to a `pin_compatible()`? Should all pins be updated as such?

Reply:
I prefer pin_compatible because it means you only have to update (and research build vs runtime compatibility) once, personally. There's not a distro-wide standard for it though.

Reply:
Since `libpq` has a `run_exports` for itself, should this just be `- libpq  # exact pin handled through libpq run_exports`?

Reply:
oh, yeah. could do that. or just leave it out. according to taste/mood/time of day.

Reply:
This is the type of thing that makes me want to spend some time on the linter. I feel like this should be something standard across our recipes as a practice. It would also be nice if that practice included a means of determining when a package that exists in build/host has a `run_exports` without having to go look at the recipe or package metadata.